app/views/collections/show.html.erb

```diff
@@ -26,11 +26,11 @@
     <h3 class="collection_revision_header">Collection Revisions</h3>
     <% @collection_records.each_with_index do |record, index| %>
 
-      <div class="collection_revision">
+      <div id="collection_revision_<%= record.revision_id %>" class="collection_revision">
         <table cellpadding="5" border="1">
 
           <tr>
-            <th>
+            <th class="collection_granule_revision_row">
```

Comment:
Now uses css that fixes the size of the header, makes it a little more aligned and consistent.